MSAL/test/unit/utils/MSALTestCase.m

```diff
@@ -60,7 +59,6 @@ - (void)setUp
 - (void)tearDown
 {
     // Put teardown code here. This method is called after the invocation of each test method in the class.
-    XCTAssertFalse([MSALWebUI cancelCurrentWebAuthSession]);
```

Comment:
we should still cancel the session in the teardown I think

Reply:
Sure